mono/mini/interp/transform.c

```diff
@@ -3787,12 +3787,10 @@ generate (MonoMethod *method, RuntimeMethod *rtm, unsigned char *is_bb_start, Mo
 				td.sp -= 3;
 				td.ip += 1;
 				break;
-#if 0
 			case CEE_NO_:
 				/* FIXME: implement */
```

Comment:
Right, it's only meant as an optimization hint for code where you're confident you know what you're doing. I figured we should just handle and ignore the instruction, rather than abort on it.